### PR TITLE
Sample Test Page UI Enhancements

### PR DESCRIPTION
**Pull request by sirknightj**
*Issue #, if available:*

*Description of changes:*
* Region dropdown
Screenshot 2023-04-27 at 2 28 00 PM

* Region validation
Screenshot 2023-04-27 at 2 29 34 PM

* Missing required fields validation
Screenshot 2023-04-27 at 2 32 23 PM

* Tooltips (work in progress)
Screenshot 2023-04-27 at 2 30 23 PM

* Errors for WebRTC Ingestion without audio/video both selected. Also move the WebRTC Storage section next to the Track selection area as those are related.
image

* Create channel button scrolls down to let you see the logs
* Add log statements when Send DataChannel Message buttons are clicked
* Clear the DataChannel message textarea when the send button is clicked

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/app.js [hunk 1]
```diff
@@ -8,8 +8,13 @@ function configureLogging() {
             .map(message => {
                 if (message instanceof Error) {
                     const { stack, ...rest } = message;
+                    if (!rest) {
+                        console.debug('aaaa', message);
+                        return stack;
+                    }
                     return `${JSON.stringify(rest, null, 2)}\n${stack}`;
                 } else if (typeof message === 'object') {
+                    console.debug('aaaaaaa', message);
```
Comment (niyatim23) [inline examples/app.js:17]: Do we want to remove these console messages?

Comment (hassanctech) [reply examples/app.js:17]: +1

Comment (sirknightj) [reply examples/app.js:17]: Great catch! Accidentally left those there

**Review by hassanctech — APPROVED** [review]
lgtm

**Review by niyatim23 — APPROVED** [review]
Nice work!

**Pull request merged**